guides/regular-releases.md

```diff
@@ -6,20 +6,21 @@ menuInclude: no
 menuTopTitle: Guides
 ---
 
-The main set of FOLIO modules are coordinated to form a regular release.
+The main set of FOLIO modules are co-ordinated to form a regular release.
 At this stage it happens on a quarterly basis.
 
-The approximate dates and descriptive names (e.g. Edelweiss, FameFlower, Goldenrod) for upcoming releases are listed at [FOLIO Development Timeline Dates](https://wiki.folio.org/display/RPT/FOLIO+Apps+and+LDP+Release+Dates+Disambiguation+Table).
-
-Refer to the [community release notes](https://wiki.folio.org/display/COMMUNITY/Release+Notes).
-
 Refer to the [Releases](https://wiki.folio.org/display/REL/) area of the Wiki for co-ordination of the next release.
+The approximate dates and descriptive names (e.g. Edelweiss, Fameflower, Goldenrod) for upcoming releases are also listed there.
 
 The preparation for each release has a dedicated Slack [channel](/guidelines/which-forum/#slack)
 (`#releases`). There are various planning documents pinned there.
 
 The cut-off dates for module releases that are to be included, are also listed in those spreadsheets.
 
+Refer to the [community release notes](https://wiki.folio.org/display/COMMUNITY/Release+Notes)
+and important upgrade considerations.
```

Comment:
The `and important` doesn't seem to follow in the sentence for me.

Did you mean something like `Refer to the community release notes for important upgrade considerations`?

Reply:
Thanks, your suggestion is better.